packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -79,6 +93,76 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
 
   @override
   final bool shouldCodesign = true;
+
+  String get exportOptionsPlist => stringArg('export-options-plist');
+
+  @override
+  Future<FlutterCommandResult> runCommand() async {
+    if (exportOptionsPlist != null) {
+      final FileSystemEntityType type = globals.fs.typeSync(exportOptionsPlist);
+      if (type == FileSystemEntityType.notFound) {
+        throwToolExit(
+            '"$exportOptionsPlist" property list does not exist. See "man xcodebuild" for available keys.');
```

Comment:
The second sentence here and below seem like non sequiturs? I assume you're just trying to say file x you specified doesn't exist?

Also be consistent with man xcodebuild vs xcodebuild -h

Reply:
Sorry I saw this after it was merged.  Will follow up.

Reply:
https://github.com/flutter/flutter/pull/68826